### PR TITLE
Feature: Support latest runtimes for AWS Lambda

### PR DESCRIPTION
**Pull request by SauarvD**
### Requirements

* Updating latest runtimes in UI and supporting old runtimes along with new ones in backend

### Description of the Change

* Updated UI with latest runtimes
* Updated Jenkins build pack with latest runtimes and also supporting old runtimes as well

### Benefits

* User will be able to use the latest runtimes provided by AWS

### CODE REVIEW

**Comment by devsatishm** [comment]
LGTM

**Review by devsatishm — APPROVED** [review]

**Comment by devsatishm** [comment]
@SauarvD Missing changes in create-service & services API. They should be updated to support the new runtimes.

**Review by devsatishm — CHANGES_REQUESTED** [review]
Need changes.

**Comment by SauarvD** [comment]
@devsatishm No changes are required for create-service & services API. We just need to update the allowed runtimes in the config DB. Once the stack is built, it will deploy the create-service & services API which will get updated with new runtimes

**Comment by devsatishm** [comment]
> @devsatishm No changes are required for create-service & services API. We just need to update the allowed runtimes in the config DB. Once the stack is built, it will deploy the create-service & services API which will get updated with new runtimes

@SauarvD  Almost! Looks like one of the swaggers & a config file need an update. You can search for `python3` or `nodejs10` and make sure all the locations reflect the new changes.

**Comment by SauarvD** [comment]
@devsatishm Done the changes

**Comment by SauarvD** [comment]
> > @devsatishm No changes are required for create-service & services API. We just need to update the allowed runtimes in the config DB. Once the stack is built, it will deploy the create-service & services API which will get updated with new runtimes
> 
> @SauarvD Almost! Looks like one of the swaggers & a config file needs an update. You can search for `python3` or `nodejs10` and make sure all the locations reflect the new changes.

Done

**Review by devsatishm — APPROVED** [review]
Ship it!

File: builds/jenkins-build-pack-api/Jenkinsfile [hunk 1]
```diff
@@ -722,13 +722,15 @@ def buildLambda(String runtime, String repo_name) {
   if (runtime.indexOf("nodejs") > -1) {
     sh "npm install --save"
   } else if (runtime.indexOf("java") > -1) {
-    sh "mvn package"
+    withEnv(["JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64"]) {
```
Comment (raghits) [inline builds/jenkins-build-pack-api/Jenkinsfile:725]: These changes may not be needed if we manage to upgrade jenkins.

File: builds/jenkins-build-pack-api/Jenkinsfile [hunk 2]
```diff
@@ -1165,7 +1167,9 @@ def runValidation(String runtime) {
   if (runtime.indexOf("nodejs") > -1) {
     sh "jshint *.js"
   } else if (runtime.indexOf("java") > -1) {
-    sh "java -cp ${configLoader.CODE_QUALITY.SONAR.CHECKSTYLE_LIB} com.puppycrawl.tools.checkstyle.Main -c sun_checks.xml src"
+    withEnv(["JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64"]) {
```
Comment (raghits) [inline builds/jenkins-build-pack-api/Jenkinsfile:1170]: These changes may not be needed if we manage to upgrade jenkins.

File: builds/jenkins-build-pack-function/Jenkinsfile [hunk 3]
```diff
@@ -695,14 +695,16 @@ def buildFunction(String runtime, String repo_name) {
   if (runtime.indexOf("nodejs") > -1) {
     sh "npm install --save"
   } else if (runtime.indexOf("java") > -1) {
-    sh "mvn package"
+    withEnv(["JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64"]) {
```
Comment (raghits) [inline builds/jenkins-build-pack-function/Jenkinsfile:698]: These changes may not be needed if we manage to upgrade jenkins.

File: builds/jenkins-build-pack-function/Jenkinsfile [hunk 4]
```diff
@@ -1147,7 +1149,9 @@ def runValidation(String runtime) {
   if (runtime.indexOf("nodejs") > -1) {
     sh "jshint *.js"
   } else if (runtime.indexOf("java") > -1) {
-    sh "java -cp ${configLoader.CODE_QUALITY.SONAR.CHECKSTYLE_LIB} com.puppycrawl.tools.checkstyle.Main -c sun_checks.xml src"
+    withEnv(["JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64"]) {
```
Comment (raghits) [inline builds/jenkins-build-pack-function/Jenkinsfile:1152]: These changes may not be needed if we manage to upgrade jenkins.

File: builds/jenkins-build-pack-sls-app/Jenkinsfile [hunk 5]
```diff
@@ -582,15 +582,17 @@ def buildSlsApp(String runtime, String repo_name) {
   if (runtime.indexOf("nodejs") > -1) {
     sh "npm install --save"
   } else if (runtime.indexOf("java") > -1) {
-    sh "mvn package"
+    withEnv(["JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64"]) {
```
Comment (raghits) [inline builds/jenkins-build-pack-sls-app/Jenkinsfile:585]: These changes may not be needed if we manage to upgrade jenkins.

File: builds/jenkins-build-pack-sls-app/Jenkinsfile [hunk 6]
```diff
@@ -854,7 +856,9 @@ def runValidation(String runtime) {
   if (runtime.indexOf("nodejs") > -1) {
     sh "jshint *.js"
   } else if (runtime.indexOf("java") > -1) {
-    sh "java -cp ${configLoader.CODE_QUALITY.SONAR.CHECKSTYLE_LIB} com.puppycrawl.tools.checkstyle.Main -c sun_checks.xml src"
+    withEnv(["JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64"]) {
```
Comment (raghits) [inline builds/jenkins-build-pack-sls-app/Jenkinsfile:859]: These changes may not be needed if we manage to upgrade jenkins.

**Review by raghits — COMMENTED** [review]
Other changes look good.

**Review by devsatishm — APPROVED** [review]

**Review by raghits — DISMISSED** [review]
LGTM

**Review by raghits — APPROVED** [review]
LGTM

**Review by devsatishm — APPROVED** [review]

**Pull request merged**